src/scripts/item-review/item-review.component.js

```diff
@@ -0,0 +1,51 @@
+import angular from 'angular';
+import template from './item-review.html';
+import './item-review.scss';
+
+function ItemReviewController($rootScope, dimSettingsService) {
+  'ngInject';
+
+  const vm = this;
+  vm.canReview = dimSettingsService.allowIdPostToDtr;
+  vm.expandReview = false;
+  vm.submitted = false;
+  vm.hasUserReview = vm.item.userRating;
+
+  vm.procon = false; // TODO: turn this back on..
+  vm.aggregate = {
+    pros: ['fast', 'lol'],
+    cons: ['ok']
+  };
+//  vm.item.writtenReviews.forEach((review) => {
+//    aggregate.pros.push(review.pros);
+//    aggregate.cons.push(review.cons);
+//  });
+
+  vm.toggleEdit = function() {
+    vm.expandReview = !vm.expandReview;
+  };
+
+  vm.submitReview = function() {
+    $rootScope.$broadcast('review-submitted', vm.item);
+    vm.expandReview = false;
+    vm.submitted = true;
+  };
+
+  vm.setRating = function(rating) {
+    if (rating) {
+      vm.item.userRating = rating;
+    }
+    vm.expandReview = true;
+  };
+}
+
+var ItemReviewComponent = {
+  bindings: {
+    item: '<'
+  },
+  controller: ItemReviewController,
+  template: template
+};
+
+angular.module('dimApp')
+  .component('dimItemReview', ItemReviewComponent);
```

Comment:
Check out the vendors.module.js for the pattern we're starting to move towards (I think) for modules-by-feature.